lib/ex.coffee

```diff
@@ -32,21 +32,19 @@ trySave = (func) ->
 
   deferred.promise
 
-saveAs = (filePath) ->
-  editor = atom.workspace.getActiveTextEditor()
-  fs.writeFileSync(filePath, editor.getText())
```

Comment:
Yes, but that is actually how vim behaves. Unless only using `:w` without arguments, in which case `editor.save` would be the go-to function.